model-optimizer/mo/utils/cli_parser_test.py

```diff
@@ -355,6 +355,20 @@ def test_mean_do_not_match_input(self):
         mean_values = parse_tuple_pairs("input_not_present(255),input2(255)")
         self.assertRaises(Error, get_mean_scale_dictionary, mean_values, scale_values, "input1,input2")
 
+    def test_values_match_input_name(self):
+        # before fix cli_parser was confused when input had a substring that matches scale/mean values
+        res_values = parse_tuple_pairs("input255(255),input255.0(255.0),multi-dotted.input.3.(255,128,64)")
+        exp_res = {'input255': np.array([255.0, 254.]),
```

Comment:
Why do we have 254.0 and how do the tests pass?

Reply:
Thanks for noting this. Agree.